site/app/templates/polls/PollForm.twig

```diff
@@ -202,7 +222,7 @@
         }
         submitErrorChecks();
     };
-
+    
```

Comment:
This whitespace change should be reverted.

Reply:
Fixed